Signal/src/call/PeerConnectionClient.swift

```diff
@@ -227,15 +262,17 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
     public func setCameraSource(useBackCamera: Bool) {
         SwiftAssertIsOnMainThread(#function)
 
-        PeerConnectionClient.signalingQueue.async {
-            guard let localVideoSource = self.localVideoSource else {
-                owsFail("\(self.logTag) in \(#function) localVideoSource was unexpectedly nil")
+        let handle = self.handle
+        PeerConnectionClient.signalingQueue.async { [weak self] in
```

Comment:
I'm still using `[weak self] in` everywhere to ensure that accidental references to `self` cause build errors.  Unfortunately, because we're not using `[weak self]` in these blocks, these cause build warnings.  I think it's worth it - it's easy to accidentally capture `self` while moving code around.